### PR TITLE
change var names to follow naming conventions

### PR DESCRIPTION
**Pull request by manjitkumar**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -33,6 +33,8 @@ jenkins_init_changes:
 
 setup_docker: false
 setup_packer: false
+jenkins_setup_docker: false
+jenkins_setup_pakcer: false
```
Comment (snarkyChef-zz) [inline defaults/main.yml]: Spelt wrong.

File: defaults/main.yml [hunk 2]
```diff
@@ -33,6 +33,8 @@ jenkins_init_changes:
 
 setup_docker: false
 setup_packer: false
```
Comment (snarkyChef-zz) [inline defaults/main.yml]: Maybe these variables can be commented out here?

File: tasks/main.yml [hunk 3]
```diff
@@ -69,7 +69,7 @@
 - include: plugins.yml
 
 - include: docker.yml
-  when: setup_docker == true
+  when: setup_docker == true or jenkins_setup_docker == true
 
 - include: packer.yml
-  when: setup_packer == true
+  when: setup_packer == true or jenkins_setup_pakcer == true
```
Comment (snarkyChef-zz) [inline tasks/main.yml]: Spelt wrong

Comment (manjitkumar) [reply tasks/main.yml]: fixed

**Comment by snarkyChef-zz** [comment]
Approved

**Pull request merged**